FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -30,6 +30,19 @@ extern NSString *const _Nonnull FIRNamespaceGoogleMobilePlatform NS_SWIFT_NAME(
 extern NSString *const _Nonnull FIRRemoteConfigThrottledEndTimeInSecondsKey NS_SWIFT_NAME(
     RemoteConfigThrottledEndTimeInSecondsKey);
 
+/**
+ * Represents a listener that can be removed by calling it's only method remove. This is returned
+ * when calling addOnConfigUpdateListener and the method should be used when you no longer want to
+ * listen for new config updates. If this is the last listener it will close the Realtime stream.
+ */
+extern @interface FIRConfigUpdateListenerRegistration : NSObject
```

Comment:
should you add a NS_SWIFT_NAME here? not sure if FIR will be automatically removed. Please add API test or swift test app so we can see how all the swift APIs are called.

Reply:
Added tests to APITests.swift for fake console and real console. Also added Swift name for registration